javascript/javascript/Cpp/JavaScriptParserBase.cpp

```diff
@@ -24,7 +24,7 @@ bool JavaScriptParserBase::next(std::string str)
 
 bool JavaScriptParserBase::notLineTerminator()
 {
-    return !here(JavaScriptParser::LineTerminator);
+    return !lineTerminatorAhead();
```

Comment:
Please remove all `here()` method definitions. You removed the applied usages for `here()`, so it does not make any sense to keep the old code around.

Reply:
Thank you, I removed the unused method. I wouldn't have figured out what was the problem with the go parser.